app/webpacker/components/Tickets/TicketWorkbenches/AnonymizationTicketWorkbenchForWrt/AnonymizeAction.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Button, Confirm, Message } from 'semantic-ui-react';
+import useSaveAction from '../../../../lib/hooks/useSaveAction';
+import Loading from '../../../Requests/Loading';
+import { actionUrls } from '../../../../lib/requests/routes.js.erb';
+
+export default function AnonymizeAction({ userId, wcaId }) {
+  const [confirmOpen, setConfirmOpen] = useState(false);
+  const [completed, setCompleted] = useState(false);
+
+  const { save, saving } = useSaveAction();
```

Comment:
I think you can use Tanstack mutation here

Reply:
Done